WeakAurasOptions/AuthorOptions.lua

```diff
@@ -975,6 +975,35 @@ typeControlAdders = {
       step = 1
     }
   end,
+  media = function(options, args, data, order, prefix, i)
+    local option = options[i]
+    args[prefix .. "mediaType"] = {
+      type = "select",
+      width = WeakAuras.normalWidth,
```

Comment:
The list isn't wide enough for anything but sounds, can you make this doubleWidth?